specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/StaticSites.json

```diff
@@ -1449,6 +1449,14 @@
         "buildProperties": {
           "$ref": "#/definitions/StaticSiteBuildProperties",
           "description": "Build properties to configure on the repository."
+        },
+        "privateEndpointConnections": {
+          "description": "Private endpoint connections referencing this resource.",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/PrivateEndpointConnectionResource"
+          },
+          "readOnly": true
         }
```

Comment:
if staticSites supports privateEndpointConnections it should expose them as child resources as well and have a privateLinkResources collection GET as a child